### PR TITLE
add vdm

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@mwilson8 @oblute agree to be co-maintainers?

**Comment by rluria14** [comment]
@mwilson8 @oblute agree to be maintainers?

File: recipes/vdm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65b75bb342280114e3c9794fbe5c54b77a9d997fbe07dae50aa303381ccfa26e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
```
Comment (scopatz) [inline recipes/vdm/meta.yaml]: Can this be `noarch: python`?

Comment (rluria14) [reply recipes/vdm/meta.yaml]: yep, noarch: python added.

**Pull request merged**